src/tests/unit/recipeService/createRecipe.test.ts

```diff
@@ -0,0 +1,41 @@
+import { IRecipe } from '../../../interfaces/recipe';
+import { Recipe } from '../../../models/recipeModel';
+import RecipeService from '../../../services/recipeService';
+
+import { mockBody, mockId, mockImage, mockTitle } from '../../mocks/mockRecipe';
+
+import { setupTests } from '../../setupTests';
+
+const recipeService = new RecipeService();
+
+setupTests('createRecipe', () => {
+  test('save new Recipe in a database', async () => {
+    await recipeService.createRecipe(mockId, mockTitle, mockImage, mockBody);
+    const recipe = <IRecipe>await Recipe.findOne({ title: mockTitle });
```

Comment:
`await recipeService.createRecipe(mockId, mockTitle, mockImage, mockBody);` should return recipe object, I know this is a test, but I would prefer to get objects from db by id.